tests/conftest.py

```diff
@@ -140,3 +142,41 @@ def onnx_dir(request):
 @pytest.fixture(scope="module")
 def ov_config_dir(request):
     return request.config.getoption("--ov-config-dir")
+
+
+@pytest.fixture(scope="function")
+def create_venv(install_type, tmp_path, package_type, venv_type):  # pylint:disable=redefined-outer-name
```

Comment:
Either remove the NNCF installation step from this fixture, or rename the fixture to `tmp_venv_with_nncf`

Reply:
Done